### PR TITLE
Fix: profile picture error persists after changing person Id

### PR DESCRIPTION
**Pull request by SketchBookkeeper**
As I have been working with this, I noticed that if the element has an error and then the person Id is changed, its still `hasError` so the new person's image will not load

### CODE REVIEW

**Comment by aarondrabeck** [comment]
``` src=${this._computeSrc(this.personId, this.size, this.cacheBust, this.hasError)}```

It should already recompute the src if the personId changes.   Is there a place were you could demo to me the behavior?

**Comment by SketchBookkeeper** [comment]
> 
> 
> ` src=${this._computeSrc(this.personId, this.size, this.cacheBust, this.hasError)}`
> 
> It should already recompute the src if the personId changes. Is there a place were you could demo to me the behavior?

Yes, I'm running into this on the new CLGame report I'm making. It'll be easier to show you once that PR is ready

**Comment by SketchBookkeeper** [comment]
> 
> 
> ` src=${this._computeSrc(this.personId, this.size, this.cacheBust, this.hasError)}`
> 
> It should already recompute the src if the personId changes. Is there a place were you could demo to me the behavior?

Actually I do have somewhere else where this is happening. So if you go to my permissions by person page https://permissions.leavitt.com/person/903279/permissions and then navigate back to the list and go to your permission page you will notice you'll just see the default profile picture. I don't have a profile image so the element's `hasError` is being set but then nothing is resetting that when you go to another page.

**Comment by AbandonedRefrigerator** [comment]
> ` src=${this._computeSrc(this.personId, this.size, this.cacheBust, this.hasError)}`
> 
> It should already recompute the src if the personId changes. Is there a place were you could demo to me the behavior?

It can't recompute the src. personId changes, which does trigger computeSrc, but the first line in computeSrc is `if (hasError) { return ...}` so hasError never has the opportunity to reset.

File: packages/profile-picture/src/profile-picture.ts [hunk 1]
```diff
@@ -49,6 +49,10 @@ export class ProfilePictureElement extends LitElement {
   }
 
   updated(changedProps) {
+    if (changedProps.has('personId')) {
+      this.refresh();
```
Comment (AbandonedRefrigerator) [inline packages/profile-picture/src/profile-picture.ts:53]: @SketchBookkeeper I'm not sure we should use refresh() here. We don't actually need to increment the cache-buster. Just set hasError to false.

Comment (SketchBookkeeper) [reply packages/profile-picture/src/profile-picture.ts:53]: Agreed, I bet there is a better way to fix this. maybe just `hasErrors = false`

**Comment by aarondrabeck** [comment]
So the hasError notion is a bit flawed.  Although this would fix a person Id change, cacheBust would still have the bug.  

Lets remove the notion hasError, and just set the person Id to 0 on error``` @error=${() => (this.personId = 0)}```.  Then we could simplify to 

```  
_computeSrc(personId: number, size: number, cacheBust: number): string {
    return `https://cdn.leavitt.com/user-${personId}-${this.determineSize(size)}.jpeg${cacheBust > 0 ? `?c=${cacheBust}` : ''}`;
  }```

and of course always draw the img tag

**Pull request merged**